config/routes.rb

```diff
@@ -33,6 +33,8 @@
       get "/products" => 'amazon_products#index'
       get "/charitable_organizations" => 'charitable_organizations#index'
 
+      post "/needs" => 'needs#create'
```

Comment:
let's group this, and the `get "/needs" => 'needs#index'` into a:

```ruby
resources :needs, only: %i[index create]
```

This will make it easier to see where to add new endpoints because they're all in one place.